schema/cryptography-defs.schema.json

```diff
@@ -0,0 +1,546 @@
+              ]
+            }
+          }
+        },
+        "required": [
+          "family",
+          "variant"
+        ]
+      }
+    },
+    "ellipticCurves": {
+      "type": "array",
+      "title": "Elliptic Curves",
+      "description": "An array of elliptic curve family definitions.",
+      "items": {
+        "type": "object",
+        "title": "Elliptic Curve Family",
+        "description": "Defines an elliptic curve family and its metadata.",
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string",
+            "title": "Curve Family Name",
+            "description": "The name of the elliptic curve family."
+          },
+          "description": {
+            "type": [
+              "string",
+              "null"
+            ],
+            "title": "Description",
+            "description": "A description of the elliptic curve family."
+          },
+          "curves": {
+            "type": "array",
+            "title": "Curves",
+            "description": "List of curves in this family.",
+            "items": {
+              "type": "object",
+              "title": "Curve",
+              "description": "Defines a specific elliptic curve and its metadata.",
+              "additionalProperties": false,
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "title": "Curve Name",
+                  "description": "The name of the elliptic curve."
+                },
+                "description": {
+                  "type": [
+                    "string",
+                    "null"
+                  ],
+                  "title": "Description",
+                  "description": "A description of the elliptic curve."
+                },
+                "oid": {
+                  "type": [
+                    "string",
+                    "null"
+                  ],
+                  "title": "OID",
+                  "description": "The Object Identifier (OID) of the elliptic curve."
+                },
+                "form": {
+                  "type": "string",
+                  "title": "Form",
+                  "description": "The form of the elliptic curve.",
+                  "enum": [
+                    "Weierstrass",
+                    "Edwards",
+                    "TwistedEdwards",
+                    "Montgomery"
+                  ]
+                },
+                "aliases": {
+                  "type": "array",
+                  "title": "Aliases",
+                  "description": "List of aliases for this curve.",
+                  "items": {
+                    "type": "object",
+                    "title": "Alias",
+                    "description": "An alias for the curve.",
+                    "additionalProperties": false,
+                    "properties": {
+                      "category": {
+                        "type": "string",
```

Comment:
Could this become an enum as well? Some of the values could be "nist", "secg", "x963", and "other".